arrow2_convert/src/field.rs

```diff
@@ -107,6 +107,15 @@ impl_numeric_type_full!(i64, Int64);
 impl_numeric_type_full!(f32, Float32);
 impl_numeric_type_full!(f64, Float64);
 
+impl ArrowField for i128 {
+    type Type = i128;
+
+    #[inline]
+    fn data_type() -> arrow2::datatypes::DataType {
+        arrow2::datatypes::DataType::Decimal(32, 32)
```

Comment:
note that this is just the default. In general the user needs to specify the correct precision and scale via e.g. `.to(DataType::Decimal(10, 11))`

Reply:
Oh good point - what would you recommend for this? One idea is we could provide a datatype override for a field to override the default.

Reply:
yeap, I would recommend an override (ideally mandatory), so that people do not mistakenly use `32, 32` without thinking about it. If not possible mandatory, always present the override in the examples so people do not forget about it.